TestAppTypescript/ios/Podfile

```diff
@@ -6,7 +6,6 @@ target 'TestAppTypescript' do
   # use_frameworks!
 
   # Pods for TestAppTypescript
-  pod 'AppCenter/Push', '~> 1.13.0'
   pod 'AppCenter/Crashes', '~> 1.13.0'
```

Comment:
1.13 ?!?!
Please update the release wiki

Reply:
Here is a [bug](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/83405) for it